x-pack/solutions/observability/packages/kbn-scout-oblt/src/playwright/fixtures/worker/profiling/profiling_setup_fixture.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { test as base } from '@kbn/scout';
+import type { KbnClient } from '@kbn/test';
```

Comment:
This might cause in the future if `KbnClient` fixture interface will deviate from the one in '@kbn/test'. We did it with `esArchiver` on purpose.

I suggest to use the default type coming from the fixture:

https://github.com/elastic/kibana/blob/2b4fa99f3b2732f8548d60c243b9449a5c591f12/src/platform/packages/shared/kbn-scout/src/playwright/fixtures/scope/worker/es_archiver.ts#L38-L47

Reply:
`kbnClient` is already of `KbnClient` type from `kbn/test`:
https://github.com/elastic/kibana/blob/41b43a024d9d9634e06944552db9004ba4022e16/src/platform/packages/shared/kbn-scout/src/playwright/fixtures/scope/worker/core_fixtures.ts#L32-L51

What TS error do you see without `kbnClient as KbnClient`?